internal/capability/operator_install_test.go

```diff
@@ -0,0 +1,90 @@
+package capability
+
+import (
+	"context"
+	"os"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/opdev/opcap/internal/operator"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	operatorv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Operator install tests", func() {
+	os.Chdir(os.TempDir())
```

Comment:
You will want to capture the TempDir, and do 

```
tmpDir, err := os.MkdirTemp("", "operator-install-*")
Expect(err).ToNot(HaveOccurred())
DeferCleanup(os.RemoveAll, tempDir)
os.Chdir(tmpDir)
```

And should be inside of a `BeforeEach(func(){})`

Reply:
Oh, and before the `os.Chdir(tmpDir)` add

```
cwd, err := os.Getwd()
Expect(err).ToNot(HaveOccurred())
DeferCleanup(os.Chdir, cwd)
```